3684/1/top_gems.rb

```diff
@@ -0,0 +1,5 @@
+require_relative 'gem_searcher'
+
+searcher = GemSearch.new('gems')
```

Comment:
It looks weird that GemSearch also responds to rate gems.
I expect that `GemSearch` does search only and some other class which sort gems depending on passed params